python/tvm/relay/frontend/tflite.py

```diff
@@ -1695,10 +1695,9 @@ def convert_fully_connected(self, op):
             raise ImportError("The tflite package must be installed")
 
         input_tensors = self.get_input_tensors(op)
-        assert len(input_tensors) >= 2, "input tensors length should be >= 2"
+        assert len(input_tensors) in (2, 3), "input tensors length should be two or three"
```

Comment:
input_tensors can now have 3 elements, but I can't see the 3rd tensors (input_tensors[2]) being used anywhere?

Reply:
line 1756. Related to bias, if it is provided.

Reply:
So is this a fix, or is it related to adding support for constant input?

Reply:
Providing that assertion was made stricter, this is a small improvement.